modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItSqlApiBaseTest.java

```diff
@@ -826,6 +826,48 @@ public void resultSetCloseShouldFinishImplicitTransaction() {
         assertEquals(0, txManager().pending(), "Expected no pending transactions");
     }
 
+
+    @Test
+    public void runScriptThatCompletesSuccessfully() {
+        IgniteSql sql = igniteSql();
+
+        try (Session session = sql.createSession()) {
+            executeScript(session,
+                    "CREATE TABLE test (id INT PRIMARY KEY, step INTEGER); "
+                            + "INSERT INTO test VALUES(1, 0); "
```

Comment:
Let's add START TRANSACTION before INSERT and COMMIT after last UPDATE.

Reply:
Updated.